lang/python/python-typing-extensions/Makefile

```
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2021 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-typing-extensions
```

```suggestion
PKG_NAME:=python-typing-extensions
```

It should match the folder name

---

Thanks, fixed